common/OpTestHMC.py

```diff
@@ -245,6 +246,30 @@ def wait_system_state(self, exp_state=OpManagedState.OPERATING, timeout=WAITTIME
             if count > 60:
                 raise OpTestError("Time exceeded for reaching %s" % exp_state)
 
+    def is_lpar_in_server(self, tgt_mg_system=None):
+        lpar_lst = self.ssh.run_command(
+                   'lssyscfg -r lpar -m %s -F name' % tgt_mg_system)
+        if self.lpar_name in lpar_lst:
+            log.info("%s lpar found in managed system %s" % (tgt_mg_system, self.lpar_name))
+            return True
+        return False
+
+    def migrate_lpar(self, src_mg_system=None, det_mg_system=None):
```

Comment:
`s/det/dest/`

Reply:
Also why are you adding this function here? It looks like this is specific to LPM . If so why are move this function to `OpTestLPM.py` ?

Reply:
In case if the parameter is not mentioned in function call, I will check for default value and fail the test.  This is a HMC related function, hence added to common/OpTestHMC.py. Where as OpTestLPM.py is a test which is part of testcases folder.  the function can be used by any one else who need to do a migration.